server/xpub/entities/manuscript/resolvers.test.js

```diff
@@ -190,11 +190,19 @@ describe('Submission', () => {
       )
 
       const storedManuscript = await Manuscript.find(id, userId)
+
+      // check the object returned is what is expected, less the status
+      const expected = lodash.cloneDeep(expectedManuscript)
+      delete expected.status
+
       expect(storedManuscript).toMatchObject({
-        ...expectedManuscript,
-        // TODO this might cause a race condition
-        status: Manuscript.statuses.MECA_EXPORT_SUCCEEDED,
+        ...expected,
       })
+
+      // Now check status. Becaue of the race condition, check either valid value
+      expect(storedManuscript.status).toEqual(
+        expect.stringMatching(/^MECA_EXPORT_SUCCEEDED|^MECA_EXPORT_PENDING/),
```

Comment:
You should be able to just put this line inside the `toMatchObject` expectation. Avoids cloning and deleting properties.

Reply:
I mean:

```
      expect(storedManuscript).toMatchObject({
        ...expectedManuscript,
        status: expect.stringMatching(/^MECA_EXPORT_(SUCCEEDED|PENDING)/),	
      })
```